api/pyproject.toml

```diff
@@ -21,6 +21,7 @@ marshmallow-dataclass = {extras = ["enum", "union"], version = "^8.5.8"}
 marshmallow = "^3.18.0"
 pip-licenses = "^4.3.2"
 gunicorn = "^21.2.0"
+setuptools = "^68.2.2"
```

Comment:
Where did this come from and why do we need it?? I don't see it in the template repo

Reply:
hmm, good question. It got added when troubleshooting, not sure if it's necessary 🤔

Reply:
oki, maybe we can remove this one then